### ISSUE TITLE
Channelizing interfaces throws exceptions

### ISSUE THREAD

**Issue by DamianZaremba** [Juniper/py-junos-eznc#816]
Attempting to configure an interface speed on a QFX switch that requires channelizing other interfaces creates an exception when using load_config.

Test case:
```from jnpr.junos import Device
from jnpr.junos.utils.config import Config

device = Device(host='192.168.0.1,
                user='valid user',
                password='some password',
                port=22,
                gather_facts=False)
device.open()
device.bind(cu=Config)
device.cu.load('set chassis fpc 0 pic 0 port 30 channel-speed 10g', format='set')
```

Exception:

```
Traceback (most recent call last):
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/device.py", line 710, in execute
    ignore_warning=ignore_warning)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/decorators.py", line 170, in wrapper
    raise ex
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/decorators.py", line 116, in wrapper
    rsp = function(self, *args, **kwargs)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/device.py", line 1255, in _rpc_reply
    return self._conn.rpc(rpc_cmd_e)._NCElement__doc
  File "/opt/python/3.6/lib/python3.6/site-packages/ncclient/manager.py", line 172, in wrapper
    return self.execute(op_cls, *args, **kwds)
  File "/opt/python/3.6/lib/python3.6/site-packages/ncclient/manager.py", line 232, in execute
    raise_mode=self._raise_mode).request(*args, **kwds)
  File "/opt/python/3.6/lib/python3.6/site-packages/ncclient/operations/third_party/juniper/rpc.py", line 44, in request
    return self._request(rpc)
  File "/opt/python/3.6/lib/python3.6/site-packages/ncclient/operations/rpc.py", line 337, in _request
    raise self._reply.error
ncclient.operations.rpc.RPCError: 
Ports 30 31 32 will be channelized or dechannelized


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/utils/config.py", line 439, in try_load
    **rpc_xattrs)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/rpcmeta.py", line 266, in load_config
    return self._junos.execute(rpc, ignore_warning=ignore_warning)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/decorators.py", line 76, in wrapper
    return function(*args, **kwargs)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/decorators.py", line 31, in wrapper
    return function(*args, **kwargs)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/device.py", line 730, in execute
    errs=ex)
jnpr.junos.exception.RpcError: RpcError(severity: warning, bad_element: None, message: Ports 30 31 32 will be channelized or dechannelized)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "test.py", line 12, in <module>
    device.cu.load('set chassis fpc 0 pic 0 port 30 channel-speed 10g', format='set')
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/utils/config.py", line 522, in load
    ignore_warning=ignore_warning)
  File "/opt/python/3.6/lib/python3.6/site-packages/jnpr/junos/utils/config.py", line 443, in try_load
    raise ConfigLoadError(cmd=err.cmd, rsp=err.rsp, errs=err.errs)
jnpr.junos.exception.ConfigLoadError: ConfigLoadError(severity: warning, bad_element: None, message: warning: Ports 30 31 32 will be channelized or dechannelized)
```

Configuring the same on the command line is indeed a warning

```
damian@lab-qfx10k2-01# set chassis fpc 0 pic 0 port 30 channel-speed 10g 
warning: Ports 30 31 32 will be channelized or dechannelized
```

However this is expected and isn't fatal.

Passing `ignore_warning=True` to the load function also doesn't appear to be supported; ignore_warning is set to True at https://github.com/Juniper/py-junos-eznc/blob/master/lib/jnpr/junos/utils/config.py#L396, but this doesn't affect the error.

I believe this warning should be excluded by default, or at least ignore_warning supporting the exclusion of it.

**Comment by stacywsmith** [comment]
What version of PyEZ are you using?

You should be able to use:

device.cu.load('set chassis fpc 0 pic 0 port 30 channel-speed 10g', format='set', ignore_warning=True)
or
device.cu.load('set chassis fpc 0 pic 0 port 30 channel-speed 10g', format='set', ignore_warning='will be channelized or dechannelized')

However, I suspect you are using PyEZ version <= 2.1.2 and that you are hitting https://github.com/Juniper/py-junos-eznc/pull/712 <https://github.com/Juniper/py-junos-eznc/pull/712> when you attempt to use ignore_warning. This is fixed in 2.1.3 and later.

—Stacy

**Comment by DamianZaremba** [comment]
It seems I was indeed hitting #712, rebuilding with 2.17 resolved the issue (this was meant to already be on this box, but apparently didn't quite make it).

```
ignore_warning=[
                # https://github.com/Juniper/py-junos-eznc/issues/816
                'will be channelized or dechannelized'
            ])
```

Allows the config to be applied as expected.

Thanks for the prompt response!
